py_vsys/contract/atomic_swap_ctrt.py

```diff
@@ -429,7 +428,7 @@ async def maker_lock(
                 data_stack=de.DataStack(
                     de.Amount.for_tok_amount(amount, unit),
                     de.Addr(md.Addr(recipient)),
-                    de.Bytes(md.Bytes(puzzle_bytes)),
+                    de.Bytes(md.Bytes(hash_secret)),
```

Comment:
```suggestion
                    de.Bytes.from_bytes(hash_secret),
```
try to use the existing function instead of the constructor.